### ISSUE TITLE
Exec Format Error in Example Notebook 1

### ISSUE THREAD

**Issue by richard-chen169** [daquinterop/Py_DSSATTools#30]
I am on an iMac (version 12.7.2) with a 4 GHz Quad-Core Intel Core i7. I installed Py_DSSATTools using pip in a virtual environment and tried running through the [Example 1 (Basics)](https://github.com/daquinterop/DSSATTools_notebooks/blob/main/Example%201%20(Basics).ipynb) python notebook.
All of the code runs fine until code block 15, where the following code is run.
`dssat.run(
    soil=soil, weather=wth, crop=crop, management=man,
)`
This gives me the error which seems to be an issue with the OS compatibility.
> OSError: [Errno 8] Exec format error: '/Users/richard/miniconda3/envs/dssat/lib/python3.12/site-packages/DSSATTools/static/bin/dscsm048'

Is Py_DSSATTools compatible with MacOS? If so, are there any suggestions on how to fix this issue?

**Comment by daquinterop** [comment]
Hi. Thanks for reporting the issue. Unfortunately I have not tested the library for MacOS so I don't think it'll work. To be honest I don't know if anyone has successfully compiled DSSAT on mac.

**Comment by daquinterop** [comment]
It definitively runs on Linux. It runs on Windows although I have not tested the latest release for Windows. I recommend using it on Linux. If you have a Windows machine the best option is to install WSL and run it there.

**Comment by daquinterop** [comment]
> I got this same error on my RaspberryPi 4
> 
> I guess it couldn't run on ARM cpu, i run it on x86 arch works

Yes, I have never tested it on a Raspberry, probably it won't in an ARM cpu. Thanks for sharing your experience!, it would be interesting to have this working in a ARM cpu.
